### PR TITLE
Docfix 3634

### PR DESCRIPTION
**Pull request by hermunn**
This patch is a bit wordy, but since the `NULL` string is not explained well, I think it is justified. Comments welcome.

### CODE REVIEW

File: vmod/vmod_cookie.vcc [hunk 1]
```diff
@@ -161,6 +160,38 @@ Example::
...
+		cookie.parse(req.http.Cookie);
+		if (cookie.isset("a_cookie")) {
+			set req.http.X-tmp = cookie.get("a_cookie");
+			# do something with the X-tmp header...
+		}
+	}
+	# if necessary, do something when a_cookie is not there
+	if (!req.http.X-tmp) {
+		# fallback case
+	}
```
Comment (Dridi) [inline vmod/vmod_cookie.vcc:193]: I would avoid nesting here:

```vcl
if (req.http.Cookie) {
	cookie.parse(req.http.Cookie);
}

if (cookie.isset("a_cookie")) {
	# do something with cookie.get("a_cookie")
} else {
	# fallback case
}
```

**Review by Dridi — APPROVED** [review]

**Comment by bsdphk** [comment]
OK with me.
